src/tools/immutable_object_cache/Types.cc

```diff
@@ -28,7 +28,7 @@ void ObjectCacheRequest::encode() {
 
 void ObjectCacheRequest::decode(bufferlist& bl) {
   auto i = bl.cbegin();
-  DECODE_START(1, i);
+  DECODE_START(2, i);
```

Comment:
... and pass the `struct_v` param to `decode_payload` below

Reply:
> ... and pass the `struct_v` param to `decode_payload` below

please check.